server/utils/applications/utils.ts

```diff
@@ -258,10 +258,27 @@ const firstPageOfApplicationJourney = (application: Application) => {
 }
 
 const getApplicationType = (application: Application): ApplicationType => {
+  if (application.isEsapApplication) {
+    return 'ESAP'
+  }
   if (application.isPipeApplication) {
     return 'PIPE'
   }
-  return 'Standard'
+
+  switch (application.apType) {
+    case 'esap':
+      return 'ESAP'
+    case 'mhapElliottHouse':
+      return 'MHAP (Elliott House)'
+    case 'mhapStJosephs':
+      return 'MHAP (St Josephs)'
+    case 'pipe':
+      return 'PIPE'
+    case 'rfap':
+      return 'RFAP'
+    default:
+      return 'Standard'
+  }
```

Comment:
CONFIRM: is this expected behaviour? The function is currently used for allocation and task summaries

Reply:
yes, this looks correct to me